Sources/ProjectDescription/RunAction.swift

```diff
@@ -86,7 +86,10 @@ public struct RunAction: Equatable, Codable {
         executable: TargetReference? = nil,
         arguments: Arguments? = nil,
         options: RunActionOptions = .options(),
-        diagnosticsOptions: [SchemeDiagnosticsOption] = [.mainThreadChecker],
+        diagnosticsOptions: SchemeDiagnosticsOptions = .options(
+            mainThreadCheckerEnabled: true,
+            performanceAntipatternCheckerEnabled: true
+        ),
```

Comment:
2 points:
1. how have we chosen these defaults? is it what Xcode preselects in new projects?
2. I think the default should be inside the option static initializer rather than here, so that people can add additional flags and still leverage the defaults for the others

Bonus point, If we want to make the defaults more prominents, we could rename the ones we want enabled by default to e.g. `mainThreadCheckerDisabled`, so the default would be false for all of them

Reply:
I respected the defaults that existed before, but let me check what are the ones that Xcode creates and stick to those in the initializer.

Reply:
oh, the diff made me think that we were changing the default here :)